web-ng/src/app/components/main-page/main-page.component.html

```diff
@@ -20,7 +20,7 @@
   <div class="content">
     <mat-sidenav-container class="main-page-container">
       <mat-sidenav-content class="mat-sidenav-content">
-        <ground-map></ground-map>
+        <ground-map (click)="sideNavOpened = false;"></ground-map>
```

Comment:
Hey, that was easier than I thought.. cool!

Reply:
By the way, I assume clicks on map markers still work? (i.e. subcomponent events get intercepted?)

Reply:
Sorry, I totally overlooked it and thought it was "[Map] Clicking on the map should deselect side panel" and did that. Just added a new commit to deselect selected feature. Please review.

Reply:
Ah, I hadn't had my morning coffee yet and also completely overlooked that collapsing the side panel doesn't deselect the feature! Thanks for the fix!

Does clicking on map markers still work with this fix?

Reply:
I reverted this change. Because with this, clicking on the markers would close the side panel. So I made this PR focus just on deselecting selected feature.

Reply:
I saw that, thanks! My question is whether registering the click event on the map component also traps clicks on map markers, in which case click events on markers themselves would not get triggered. Have you tried that and it still works?

Reply:
Yes, I've tried. If you register the click event on the map component, clicking the markers will trigger the event.

But if you use (mapClick) like I used in the other file on the map, it means map excluding the markers (and I guess any other components as well).